src/Common/src/Interop/Windows/winhttp/Interop.SafeWinHttpHandle.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class WinHttp
+    {
+        // TODO: Move SafeHandleZeroOrMinusOneIsInvalid class to Common
```

Comment:
@CIPop I'm keeping the TODO comment.  The issue about moving the SafeHandleZeroOrMinusOneIsInvalid is being re-considered and part of #2165